apps/ui/src/components/Ui/Button.vue

```diff
@@ -36,6 +36,23 @@ withDefaults(
 
   &.primary {
     @apply bg-skin-link text-skin-bg border-skin-link;
+
+    &:disabled {
```

Comment:
```suggestion
    &:disabled:deep() {
```

With this we can add back `scoped` attribute.